src/network/constants.rs

```diff
@@ -52,6 +53,13 @@ pub fn magic(network: Network) -> u32 {
     }
 }
 
+/// Lowest possible difficulty for Mainnet.
```

Comment:
these constants should be in consensus_params  and not be global. if they were global then what is the use of collecting them into parameter objects?